databricks/koalas/frame.py

```diff
@@ -2145,6 +2145,159 @@ def sort_values(self, by, ascending=True, inplace=False, na_position='last'):
         else:
             return kdf
 
+    # TODO: Not Supported:  keep = First
+    def nlargest(self, n, columns, keep='first'):
+        """
+        Return the first `n` rows ordered by `columns` in descending order.
+        Return the first `n` rows with the largest values in `columns`, in
+        descending order. The columns that are not specified are returned as
+        well, but not used for ordering.
+        This method is equivalent to
```

Comment:
this is not true, in Koalas the performance will be the same. So we should just add a note section saying the

```
"In pandas, ``nlargest(n)`` is more performant than ``sort_values(columns, ascending=False).head(n)``. In Koalas, thanks to Spark's lazy execution and query optimizer, the two would have same performance.
```